docs/guide/acquiring_data/database.rst

```diff
@@ -800,3 +791,196 @@ to 10 and therefore removes the 5 entries that been used least recently.
      18    2011-06-07 20:37:52 ...               19.5               19.5
      58    2011-06-07 06:33:29 ... 17.400000000000002 17.400000000000002
      59    2011-06-06 00:00:00 ...                N/A                N/A
+****************************************************************************************************************************************
```

Comment:
Let's add a file to a new file, and the name of file is 'QueryingHelioviewer.org_with_sunPy'  .
which is helped for 'SunPy can be used to make several basic requests using the The Helioviewer.org API including generating a PNG screenshot and downloading a JPEG 2000 image.'  likely related problem.

Reply:
What related problem?

We already have all of this text in the helioviewer.rst file, so I am still confused as to its inclusion here.

Reply:
here my mistake ,sorry.

Reply:
That is no problem. If you do need any help or advice, you can ask!